corehq/warehouse/models/staging.py

```diff
@@ -74,6 +76,49 @@ def raw_record_iter(cls, start_datetime, end_datetime):
         return iter_docs(Group.get_db(), group_ids)
 
 
+class UserStagingTable(StagingTable):
+    slug = 'user_staging'
+
```

Comment:
i think it would be good to bring in base_doc on the staging table, and transform to a deleted flag on the dim